jumanji/environments/combinatorial/routing/rb_board_generator_v1.py

```diff
@@ -0,0 +1,319 @@
+#from constants import EMPTY, OBSTACLE, SOURCE, TARGET, HEAD, NOOP, LEFT, LEFT, UP, RIGHT, DOWN
+from constants import SOURCE, TARGET, HEAD
+from dataclasses import dataclass
+import numpy as np
+from copy import deepcopy
+import random
+WIRE = SOURCE
+
+@dataclass
+class Vector:
```

Comment:
I just noticed that they have basically the same thing except it's called position in the types.py file. Just worth a double check because it inherits from NamedTuple and I'm not 100% sure why. 
Not sure if we want to use Position or our dataclass. Vector is probably fine for now but let's keep this in mind